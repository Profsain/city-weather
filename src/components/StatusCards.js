import PropTypes from 'prop-types';
import React from 'react';
import '../styles/StatusCard.css';

const StatusCards = ({ title, measure, unit }) => (
  <div className="StatusCard">
    <h4 className="Title">{title}</h4>
    <p className="Record">
      {measure}
      <span>{unit}</span>
    </p>
  </div>
);

StatusCards.propTypes = {
  measure: PropTypes.number,
  title: PropTypes.string,
  unit: PropTypes.string,
};

export default StatusCards;
