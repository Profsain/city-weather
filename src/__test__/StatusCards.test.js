import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import StatusCards from '../components/StatusCards';

it('Renders StatusCards component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <StatusCards />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
