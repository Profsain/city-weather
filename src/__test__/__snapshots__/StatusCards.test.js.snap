// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders StatusCards component correctly 1`] = `
<div
  className="StatusCard"
>
  <h4
    className="Title"
  />
  <p
    className="Record"
  >
    <span />
  </p>
</div>
`;
